tests/Unit/Visualization/Python/Test_PlotTrajectories.py

```diff
@@ -0,0 +1,143 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import os
+import shutil
+import unittest
+
+import numpy as np
+from click.testing import CliRunner
+
+import spectre.IO.H5 as spectre_h5
+from spectre.Informer import unit_test_build_path, unit_test_src_path
+from spectre.Visualization.PlotTrajectories import plot_trajectories_command
+
+
+class TestPlotTrajectories(unittest.TestCase):
+    def setUp(self):
+        self.test_dir = os.path.join(
+            unit_test_build_path(), "Visualization", "PlotTrajectories"
+        )
+        self.h5_filename = os.path.join(
+            unit_test_src_path(),
```

Comment:
Put this into the `self.test_dir` as well.

More info: The `unit_test_src_path` points to the source code directory. You don't want to create temporary files there, since it can mess with git.